chemicaltoolbox/rdock/select_points_SDF.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="ctb_select_points" name="Create Frankenstein ligand" version="0.1.0">
+    <description>for rDock active site definition</description>
+    <requirements>
+        <requirement type="package" version="3.7">python</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python '$__tool_directory__/select_points_SDF.py'
+        -i '$infile'
+        -o '$outfile'
+    ]]></command>
+    <inputs>
+        <param name="infile" type="data" format="sdf" label="Input file" help="Input file in SDF format"/>
+    </inputs>    
+    <outputs>
+        <data name="outfile" format="sdf"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="infile" value="input.sdf" ftype="sdf"/>
+            <output name="outfile" ftype='sdf' file="select_points_output.sdf"/>
+        </test>
+        <test>
+            <param name="infile" value="input_v3000.sdf" ftype="sdf"/>
+            <output name="outfile" ftype='sdf' file="select_points_output_v3000.sdf"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+
+.. class:: infomark
+
+**What it does**
+
+Converts an SD file containing multiple molecules into a 'Frankenstein ligand' with a 3D shape combining all ligands, which can be used for definition of an active site with rDock.
+
+This tool is based on a Perl script originally written by Peter Schmidtke.
+
+_____
+
+.. class:: infomark
+
+**Input**
+
+       - SD file containing multiple molecules.
+
+_____
+
+        
+.. class:: infomark
+
+**Output**
+
+       - SD file containing the 'Frankenstein ligand'.
+    ]]></help>
+       <citations>
+            <citation type="doi">10.1371/journal.pcbi.1003571</citation>
```

Comment:
what is this I can not find it?

Reply:
rdock citation. Strange... it works fine for me: https://doi.org/10.1371/journal.pcbi.1003571/